Sources/UInt128.swift

```diff
@@ -440,7 +440,33 @@ extension UInt128 : BinaryInteger {
     // MARK: Instance Properties
     
     public static var bitWidth : Int { return 128 }
-    
+
+
+    // MARK: Instance Methods
+
+    public var words: [UInt] {
+        guard self != UInt128.min else {
+            return []
+        }
+
+        var words: [UInt] = []
+
+        for n in 0 ... self.bitWidth / UInt.bitWidth {
```

Comment:
Would you mind changing `n` to something more descriptive, like `currentWord`? I was "forced" to stick with `n` before because that's how the Swift standard library called it and I wanted to be consistent, but I would prefer to use more descriptive names when possible.